doc/api/classes/LuaItem.luadoc

```diff
@@ -39,8 +44,12 @@ function remove() end
 --- [RW] A generic parameter.
 --  @tfield num param4
 
---- [R] The new version prototype ID of the item.
---  @tfield num new_id
+--- [R] The material ID of this item. To change it, use
+--  LuaItem.change_material.
+--  @tfield num param4
```

Comment:
There are two documentation for `param4`. Does the latter (newly added) refer `material`? `param4` is actually not item material.